### PR TITLE
Rspec 2 errors

### PR DESCRIPTION
**Pull request by perryr16**
Not sure if there is anything new in the code. So much has changed today... Either way, there are only 2 errors

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -127,15 +127,15 @@ def self.highest_scoring_visitor_team_id
   end
 #deliverable
   def self.highest_scoring_home_team_id
-    average_goals_by(:home_team).max_by{ |team_id, away_goals| away_goals}.first
+    average_goals_by(:home_team).max_by{ |team_id, home_goals| home_goals}.first
   end
```
Comment (stellakunzang) [inline lib/game.rb:131]: seems like this was a good catch?

File: lib/game_team.rb [hunk 2]
```diff
@@ -257,24 +257,29 @@ def self.opponents_wins(team_id)
...
   def self.rival_id(team_id)
+    team_id = team_id.to_i
     record_length = {}
     opponents_records(team_id).map do |team_id, record|
       record_length[team_id] = record.length
     end
     opponents = opponents_records(team_id).keys
     opponents.max_by {|opponent| (opponents_wins(team_id)[opponent].to_f / record_length[opponent].to_f).round(2)}
   end
+#
```
Comment (stellakunzang) [inline lib/game_team.rb:280]: does not freeze. does take a long time. noted. I wonder why this takes twice as long as the method above which is nearly identical.

**Comment by foleymichelle9** [comment]
Looks good to me!

**Pull request merged**